test/files/neg/t12289/c_2.scala

```diff
@@ -0,0 +1,6 @@
+//> using options -d /tmp
```

Comment:
Maybe make it a `DirectTest` and use `testOutput.path`?

Reply:
How about `-Ystop-before:jvm`. But a test for programmatic `outputDirs.outputs` would have to be `DirectTest`.

Reply:
`-Yskip:jvm` is somehow cooler. Either works. Could stop earlier, as it's just a backstop.